client/src/components/FileListItem.tsx

```diff
@@ -31,17 +47,22 @@ const FileListItem = ({
     >
       <div className='flex items-center gap-5'>
```

Comment:
You might want to put the `remove` button inside this div bc when resizing, the text and the remove button overlapped unless this is the design you want.
![image](https://github.com/davidpak/SmartNote/assets/137979931/7d6593b4-ccb7-4512-9071-c4085e5f7a4b)

Reply:
The spacing gets messed up if I move the remove button inside. The problem is that the text truncation is set to a fixed width, so it's not very flexible, but I can reduce the width so it doesn't overlap.